app/models/base_facility.rb

```diff
@@ -152,14 +152,39 @@ def find_facility_by_id(id)
 
     def query(params)
       # TODO: Sort hours similar to `find_factility_by_id` method on line 146
```

Comment:
is this TODO relevant here?  should it move closer to what it speaks to (and update referenced line number)?

Reply:
This TODO is 8 months old, and I don't understand what it is referencing even before my code changes. I very much dislike TODOs as anything other than a note to myself within the span of one commit. I'm in favor of deleting and just saying this should be a story if it's still relevant. 

@dhall Any feelings or context one way or another on this?